specification/securitycenter/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -0,0 +1,2067 @@
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Kind of the resource"
+                }
+            }
+        },
+        "Location": {
+            "type": "object",
+            "description": "Describes an Azure resource with location",
+            "properties": {
+                "location": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Location where the resource is stored"
+                }
+            }
+        },
+        "TaskList": {
+            "readOnly": true,
+            "type": "object",
+            "description": "List of security task recommendations",
+            "properties": {
+                "value": {
+                    "readOnly": true,
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Task"
+                    }
+                },
+                "nextLink": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The URI to fetch the next page."
+                }
+            }
+        },
+        "Task": {
+            "readOnly": true,
+            "type": "object",
+            "description": "Security task that we recommend to do in order to strengthen security",
+            "properties": {
+                "properties": {
+                    "$ref": "#/definitions/TaskProperties"
+                }
+            },
+            "allOf": [{
+                "$ref": "#/definitions/Resource"
+            }]
+        },
+        "TaskProperties": {
+            "readOnly": true,
+            "type": "object",
+            "description": "Describes properties of a task.",
+            "properties": {
+                "state": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "State of the task (Active, Resolved etc.)"
+                },
+                "creationTimeUtc": {
+                    "readOnly": true,
+                    "type": "string",
+                    "format": "date-time",
+                    "description": "The time this task was discovered in UTC"
+                },
+                "securityTaskParameters": {
+                    "$ref": "#/definitions/SecurityTaskParameters"
+                }
+            }
+        },
+        "SecurityTaskParameters": {
+            "readOnly": true,
+            "type": "object",
+            "description": "Changing set of properties, depending on the task type that is derived from the name field",
+            "properties": {
+                "name": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Name of the task type"
+                }
+            }
+        },
+        "OperationList": {
+            "readOnly": true,

```

Comment:
make sure to remove this from the model definition. readOnly can be specified only for properties of a model.

Reply:
removed